analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Verifier.cs

```diff
@@ -190,6 +193,37 @@ private string InsertConcurrentNamespace(string content)
                 ImportsRegexVB.Match(content) is { Success: true } match ? match.Index + match.Length + 1 : 0;
         }
 
+        private static string EncloseInNamespace(string content)
+        {
+            var tree = CSharpSyntaxTree.ParseText(content);
```

Comment:
This is overkill. Simple string manipulation could do the trick.
The additional syntax parsing is not worth the trouble.